docs/extensions-plugins/obsidian.mdx

```diff
@@ -0,0 +1,186 @@
+---
+title: Pieces for Developers Obsidian Plugin
+description: Troubleshoot issues or reach out to Pieces Support for assistance with the Pieces for Developers Obsidian Plugin
+---
+
+<link
+  rel="canonical"
+  href="https://docs.pieces.app/extensions-plugins/obsidian"
+/>
+
+export const InstallButton = (props) => {
+  return (
+    <a
+      style={{ textDecoration: "none" }}
+      target="_blank"
+      href="{{ links.extensions.obsidian }}"
+    >
+      <div style={{ maxWidth: "fit-content", minWidth: "fit-content" }}>
+        <p
+          style={{
+            padding: "8px 20px",
+            borderRadius: "8px",
+            border: "1px solid #f3f5f6",
+            color: "white",
+            fontWeight: "500",
+            background: "black",
+            marginBottom: "8px",
+            marginTop: "3px",
+            marginRight: "16px",
+          }}
+        >
+          {props.children}
+        </p>
+      </div>
+    </a>
+  );
+};
+
+export const ReleasePill = (props) => {
+  return (
+    <a
+      style={{ textDecoration: "none" }}
+      target="_blank"
+      href={"https://code.pieces.app/updates"}
+    >
+      <div style={{ maxWidth: "fit-content", minWidth: "fit-content" }}>
+        <p
+          class="dark:text-white text-black"
+          style={{
+            padding: "8px 20px",
+            borderRadius: "8px",
+            border: "1px solid #f3f5f6",
+            backgroundColor: "transparent",
+            fontWeight: "500",
+            marginBottom: "8px",
+            marginTop: "3px",
+            marginRight: "16px",
+          }}
+        >
+          {props.children}
+        </p>
+      </div>
+    </a>
+  );
+};
+
+export const GridFlexRow = (props) => {
+  return (
+    <div
+      style={{
+        display: "flex",
+        flexDirection: "row",
+        justifyContent: "start",
+        flexWrap: "wrap",
+      }}
+    >
+      {props.children}
+    </div>
+  );
+};
+
+export const HeaderLogoImage = (props) => {
+  return (
+    <img
+      style={{ marginLeft: "0px", height: "30%", marginRight: "12px" }}
+      src={"/assets/obsidian.png"}
+    />
+  );
+};
+
+<GridFlexRow>
+  <h1>Pieces Obsidian Plugin</h1>
+</GridFlexRow>
+
+<br></br>
+
+<GridFlexRow>
+  <InstallButton>Get the Obsidian Plugin</InstallButton>
+  <ReleasePill children="Current Version: {{ versions.obsidian }}"></ReleasePill>
+</GridFlexRow>
+
+<br></br>
+
+# Your Guide to Getting Started with Pieces for Obsidian
+
+_Pieces for Obsidian Pieces captures **NO identifiable user data**. Our **Local only** architecture means your notes never have to leave your device!_
+
```

Comment:
can we include the instructions for getting started with the Obsidian plugin? they are not here in this section - preferably in a step by step fashion